### PR TITLE
build: use rq

### PR DESCRIPTION
**Pull request by srenatus**
I haven't updated the readme yet -- let's first discuss if this is a direction we'd like to take 😃

### CODE REVIEW

**Comment by srenatus** [comment]
Gave it some optional structure there when run on github
image

File: build/do.rq [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+do.pull_request {
+	some x in ["test", "lint", "e2e", "check_readme"]
+	github("::group::", x)
+	job[x]
+	github("::endgroup::", x)
+}
+
+job.pr {
+	print("thanks for contributing")
+}
```
Comment (charlieegan3) [inline build/do.rq:19]: This isn't used right?

**Review by charlieegan3 — COMMENTED** [review]
This is cool! 😎 Some explanatory comments in do.rq or in some docs would be helpful I think.

**Review by anderseknert — APPROVED** [review]

File: build/do.rq (hunk 1 above)
Comment (anderseknert) [reply build/do.rq:19]: No, I'll add those steps later to help run all steps needed to prepare a PR.

**Pull request merged**